apps/obsidian/src/utils/shapes/discourseNodeShapeUtils.ts

```diff
@@ -0,0 +1,61 @@
+import type { App, TFile } from "obsidian";
+import type DiscourseGraphPlugin from "~/index";
+import type { DiscourseNode } from "~/types";
+import { resolveLinkedFileFromSrc } from "~/utils/assetStore";
+
+export type FrontmatterRecord = Record<string, unknown>;
+
+export const getLinkedFileFromSrc = async (
```

Comment:
Is the only purpose of this function to call another function?  Why not just call `resolveLinkedTFile...()`?